<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExternalCommWidget</class>
 <widget class="QWidget" name="ExternalCommWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>-1</y>
     <width>981</width>
     <height>751</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>协议设置</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>协议选择:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="selectProtocolcomboBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>35</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="ModelSelectComboBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>102</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>101</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>通讯模式:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>通讯设定:</string>
     </property>
    </widget>
    <widget class="QWidget" name="CommTypeWidget" native="true">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>831</width>
       <height>491</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>数据设置</string>
    </attribute>
    <widget class="QWidget" name="Protocol_widget" native="true">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>971</width>
       <height>701</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
